Meta/build-image-qemu.sh

```diff
@@ -20,12 +20,17 @@ SCRIPT_DIR="$(dirname "${0}")"
 
 USE_FUSE2FS=0
 
+if ! hash sudo >/dev/null 2>&1
```

Comment:
If we want to use `doas`, we should check if `doas` is present. What you are doing here completely breaks support for SerenityOS.